README.md

```diff
@@ -15,54 +15,59 @@ Now:
 * add expRESTive to your _package.json_ file: `$ npm install --save exprestive`
 * add the expRESTive middleware to your application
 
-    ```coffeescript
-    express = require 'express'
-    exprestive = require 'exprestive'
+    ```javascript
+    express = require('express')
+    exprestive = require('exprestive')
 
     app = express()
-    app.use exprestive()
+    app.use(exprestive())
 
-    app.listen 3000
+    app.listen(3000)
     ```
 
-* create a `routes.coffee` file in the same directory as your server
+* create a `routes.js` file in the same directory as your server
 
-  ```coffeescript
-  module.exports = ({ GET, POST, PUT, DELETE }) ->
-    GET '/hello',   to: 'helloWorld#index'
+  ```javascript
+  // routes.js
+  module.exports = ({ GET, POST, PUT, DELETE }) => {
+    GET('/hello', { to: 'helloWorld#index' })
+  }
   ```
 
 * create a `controllers/` directory in the same directory as your server and populate it with controllers.
   Controller file names must end in `controller` (with a js or compile-to-js extension). This restriction can be changed with the **controllersPattern** option. See [options](#options).
 
-  ```coffeescript
-  # controllers/hello_world_controller.coffee
-  class HelloWorldController
-    index: (req, res) ->
-      res.end 'hello world'
-
-  module.exports = HelloWorldController
+  ```javascript
+  // controllers/hello_world_controller.js
+  module.exports = class HelloWorldController {
+    index(req, res) {
+      res.end('hello world')
+    }
+  }
```

Comment:
I would suggest still having the module.exports at the end. Just keeps the class line to be a bit shorter and I think makes the class read a bit better.